frontend/src/app/shared/components/header/header.component.ts

```diff
@@ -11,14 +12,23 @@ import { UserNotificationComponent } from '../user-notification/user-notificatio
 export class HeaderComponent implements OnInit {
     @ViewChild('notificationsMenu') notificationsMenu: UserNotificationComponent;
 
+    currentUser: IUserInfo;
+
     constructor(private authService: AuthService) {}
 
     ngOnInit(): void {
         if (this.authService.isAuthenticated()) {
             this.authService.setUserSection();
+            const userSection = this.authService.getUserSection();
+
+            if (userSection) {
+                this.currentUser = userSection;
+            }
```

Comment:
Still nope... 
setUserSection may take some time to execute. During this time this function will be finished and we will get empty user from localstorage